lib/logstash/inputs/file.rb

```diff
@@ -23,6 +23,12 @@
 # beginning to end and storing all of it in a single event (not even
 # with the multiline codec or filter).
 #
+# ==== Reading from remote network volumes
+#
+# Currently, File input does not support reading files from remote network volumes like NFS and Samba.
```

Comment:
I'd be more explicit to say "the file input is not tested on remote filesystems such as NFS, Samba, s3fs-fuse, etc. These remote filesystems typically have behaviors that are very different from local filesystems and are therefore unlikely to work correctly when used with the file input.."